### ISSUE TITLE
Gather demographics about our food seekers

### ISSUE THREAD

**Issue by GigiUxR** [hackforla/food-oasis#1441]
We are currently lacking foundational information about one of our main users, food seekers in LA. To better understand who our food seekers are, we can begin distilling existing data on LA food seekers. 

*** This issue is under construction. I am considering to split primary and secondary research tasks for better accuracy of tasks **

- [x] #1583
- [x] Discuss issue with Stacey to find out what information she has access to: are informational interviews possible with her partnership contacts? What data can they share?
- [x] Create research plan (on-going) https://docs.google.com/document/d/1mkoCSUoHg8g2I5bSsxjwYvBaeBDx_XF0CDJUsgFjemQ/edit#
- [x] Compile a list of preliminary findings and questions  https://docs.google.com/document/d/1RnEpBgUjH1XAk0h0RB1exI6GhCX7z3DREHNLgjKIb-o/edit
- [ ] #1582
- [ ] Write up a report on demographics and meaningful subgroups of LA food seekers
- [ ] Share the report with the rest of the research team to develop next steps and a suggested plan
- [ ] Present the report and suggested plan with stakeholders to further develop a food seekers research plan 

### Child projects that help complete this one
- [ ] #1525
- [ ] #1533

**Comment by staceyrebekahscott** [comment]
Closing issue #964 Food Seekers Interviews, created in 04/2021. This new issue is more detailed and relevant to the current state of the project.

**Comment by staceyrebekahscott** [comment]
@GigiUxR As per your comments on #964, which I just closed, I wanted to bring your attention to to the fact that I noted it in this issue. Since 964 is outdated, we can use this as a beginning issue for the food seekers research, as the scope here is better defined.

**Comment by fancyham** [comment]
Since you're looking at both demographics (quantitative/ statistical stuff) and also starting to look at interviews (qualitative stuff), perhaps we can change the title of this issue to Gather demographics **+ details/learn** about our food seekers" or something more comprehensive?

Also, as you're progressing and figuring out next steps, please add your meeting notes here for posterity and for your own reference!  It's really handy to look back at past meeting notes to figure out what you considered and what you decided.  Also, gives a change for other folks to add comments and help out. :)

Also, at some point, you may have several projects that fit under this umbrella — if that's the case, don't be afraid to split that out into a separate bite-sized issue from this main one.  A separate issue can make it easier to track progress especially if you are doing things in parallel.  Ping me if you'd like to discuss.

If you make a checkbox list here or in the top issue description, then hover your mouse over it, you may see a 'create issue' button — that can be useful for doing these sub-tasks.

For example:
Goal: Make Hot Fudge Sunday
- [ ] Acquire Fudge
- [ ] Acquire Ice Cream
- [ ] Acquire banana
- [ ] Assemble Sunday

**Comment by naraedesigns** [comment]
Great points, thank you! 

------------------------------
1/11/23 - Meeting with Sarah, Jelena, Narae
Discussed issue #1441

Pain points 
- Sarah noted low response from email outreach to partners about doing informational interviews, and that people are busy doing the work of providing food resources

Key takeaways 
- We need to know - **who are our actual users?** Jelena recently did a Google Analytics presentation - current users are not really food seekers.
- Next steps to get to know our users? Sarah mentioned it might be better to interview partners first, gather useful info, then interview food seekers (most traumatized, vulnerable population). Also benefit of interviewing partners is we can publicize Food Oasis too, as well as build relationships with people/places that may be able to connect us with food seekers for research. 

Next steps:
- Continue with current action items listed on issue
- Get more contacts (LA nonprofits / partners)
- Reach out for interviews
- Create a research plan

**Comment by staceyrebekahscott** [comment]
@naraedesigns @cheesarah @jelenaUX I have been planning on reconnecting with [Food Cycle LA](https://www.foodcyclela.org/) as they are running our embed on their site. We already have contact information and I would like to work with you all to plan this outreach. Perhaps we can connect sometime next week to discuss?

**Comment by jelenaUX** [comment]
Oh great, we were just talking about starting with informational 
interviews! I am available to talk next Tuesday through Friday, from 
about 6pm-7:30pm. Also earlier any time during the day on Wednesday and 
Thursday.

On 2023-01-13 9:27 pm, Stacey Rebekah Scott wrote:

> @naraedesigns [1] @cheesarah [2] @jelenaUX [3] I have been planning on 
> reconnecting with Food Cycle LA [4] as they are running our embed on 
> their site. We already have contact information and I would like to 
> work with you all to plan this outreach. Perhaps we can connect 
> sometime next week to discuss?
> 
> --
> Reply to this email directly, view it on GitHub [5], or unsubscribe 
> [6].
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>



Links:
------
[1] https://github.com/naraedesigns
[2] https://github.com/cheesarah
[3] https://github.com/jelenaUX
[4] https://www.foodcyclela.org/
[5] 
https://github.com/hackforla/food-oasis/issues/1441#issuecomment-1382638508
[6]

**Comment by fancyham** [comment]
Also, I think this team was asking if we had any marketing-type materials we could give orgs that we’re interviewing with.  

We don’t currently but I created an issue (#1506 ) to start putting one together but it might take some time.  If you end up creating one yourself, please share it with the following issue.  That same text might help shape or replace the static text on our website’s ‘about page’ while the new text is being worked on.  Good enough is often better than waiting for perfect :)

- #1506

**Comment by staceyrebekahscott** [comment]
We can use the slide deck presentation I created for the Impact Sprints. This is intended to be used as marketing material. I made a copy of it that can be modified, as the original has details about the Brigade Congress 2022. The copy is in the main folder of the Google Drive. [Copy of Hack for LA | Food Oasis Overview Slide Deck](https://docs.google.com/presentation/d/1ljw7e_BmkE-I-9TbkYxANI1BlT-JP5uxdt-b3XpaDWA/edit#slide=id.g173838d392e_0_0). It might work for now to make this into a PDF to share if that is easier.

**Comment by cheesarah** [comment]
Thank you for creating the issue, Brian. And for sending the slide deck, Stacey.

I have been using a very generic sentence to introduce the organization (for making contact with food pantries for our demographic research), but would love to copy and paste the text in Slide #3 to introduce Food Oasis for now, and then whenever a more comprehensive intro sheet is available, I can start using that. 


Sarah Eunkyung Chee

“I am the sum total of everything that went before me, of all I have been seen done, of everything done-to-me. I am everyone everything whose being-in-the-world affected was affected by mine. I am anything that happens after I'm gone which would not have happened if I had not come.”
	
	- Salman Rushdie, Midnight’s Children

**Comment by cheesarah** [comment]
Sorry, I meant slide 2, 7 & 8. Would it help for me to put these together into a few paragraphs and send it over to the writing team? 

Sarah Eunkyung Chee

“I am the sum total of everything that went before me, of all I have been seen done, of everything done-to-me. I am everyone everything whose being-in-the-world affected was affected by mine. I am anything that happens after I'm gone which would not have happened if I had not come.”
	
	- Salman Rushdie, Midnight’s Children

**Comment by jordan-a-turner** [comment]
As part of our goal of understanding our food seekers, I’ve been looking at secondary quantitative demographic data on food seekers in LA county. So far I have looked at the LA County data from the University of Southern California's Understanding America Study.

During our food seeker research group meeting on March 3rd, Bryan shared a few demographics that could be most actionable for the design team. Based on that, here are the specific food seeker demographics I'm looking for:

- [ ] Age range—younger may be more likely to access our website via smartphones and may find information about free food resources in different ways
- [ ] Tech usage—which devices do people have/use, or which did they use to fill out the Understanding America Study survey
- [ ] Language proficiency
- [ ] Transportation availability near them
- [ ] Which foods are culturally relevant for them—many people may sell food if it is not culturally relevant and use the money to buy relevant food
- [ ] Current employment information
- [ ] Whether they qualify for/use SNAP, Meals on Wheels, etc.
- [ ] Living situation—are they seniors living alone who may not be physically able to carry large quantities of food home; how many in-home dependents
- [ ] Whether their food insecurity is short-term or long-term (I have longitudinal data)—especially whether/how it was affected by covid
- [ ] How demographics differ by area in LA county—I will be requesting access to data at the zip code level

**Links**

Google Drive document with specific insights so far: [https://docs.google.com/document/d/1pUiaYfE2uzRBonlP4TylWxUcZfGBavGpCmml2LwzUNk/edit](url) 
Information about the Understanding America Study: [https://uasdata.usc.edu/index.php](url)
Each individual who wants to look at the data has to register, so I can't link the data, but you can register here: [https://uasdata.usc.edu/index.php](url)

**Comment by fancyham** [comment]
Adding to this: I’ve seen some folks sell foods that are culturally inappropriate for them (and probably use that money to buy things that are) — but that’s in the big city and only occasionally.  Far more often, I’ve seen people pass the foods that they can’t use to their neighbors, or will tell the food distribution to give it to someone else.

But the crux is that sometimes, people don’t know what to do with or don’t want certain foods, and while some pantries allow people to pick and choose from shelves of food (the best situation), many more are really more distributions where you give everyone a big bag or box of food with a focus on speed and efficiency rather than choice.

**Comment by jordan-a-turner** [comment]
Here's the [slide deck](https://docs.google.com/presentation/d/1EojiMyb9FbQom-grVRr9IpRj-_p0UT-PPNEPgSyxBqE/edit?usp=share_link) and [video recording](https://drive.google.com/drive/folders/1rdfqiNvjofnjHzq7GL-uA3WL9n17f3fO?usp=share_link) from my presentation about LA food seeker demographics/needs in our food seeker research team meeting today.
